authorization-service/src/functions/basicAuthorizer/handler.ts

```diff
@@ -0,0 +1,56 @@
+import { middyfy } from '@libs/lambda';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { APIGatewayTokenAuthorizerEvent } from 'aws-lambda';
+import { headers } from '@libs/headers';
+
+import { generatePolicy } from '@helpers/generatePolicy';
+
+export const basicAuthorizer = async (
+    event: APIGatewayTokenAuthorizerEvent) => {
+    
+    console.log('Decoding credentials...', event);
+        const ALLOW = 'Allow';
+        const DENY = 'Deny';
+
+        try {
+
+            const { type, authorizationToken, methodArn } = event;
+
+        if (!authorizationToken || type !== 'TOKEN') {
+            console.log('Basic authorizer: No authorization token provided');
+            return formatJSONResponse({
```

Comment:
As far as I understand a Lambda authorizer function should return either _a generated policy_ or throw _an error_. So there is no need to construct HTTP-like response for it.

Here are some useful links with more info about the return values of Lambda authorizers:

- https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-output.html
- https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html